Specs/Number/German/OrdinalModel.json

```diff
@@ -507,7 +507,7 @@
   },
   {
     "Input": "Wäre unsere Sonne der jüngste Stern der Milchstraße, so wäre sie mindestens der einhundert Milliardste und höchstens der dreihundert Milliardste Stern.",
-    "NotSupported": "javascript",
+    "NotSupported": "javascript,python",
```

Comment:
Same for these 3.

Reply:
Expected result is
```
      {
        "Text": "einhundert milliardste",
        "TypeName": "ordinal",
        "Resolution": {
          "value": "100000000000",
          "offset":"100000000000",
          "relativeTo":"start"
        },
        "Start": 80,
        "End": 101
      },
      {
        "Text": "dreihundert milliardste",
        "TypeName": "ordinal",
        "Resolution": {
          "value": "300000000000",
          "offset":"300000000000",
          "relativeTo":"start"
        },
        "Start": 121,
        "End": 143
      }
```

Actual result is
```
{
	"start": 80,
	"end": 101,
	"resolution": {
		"value": "1000000100"
	},
	"text": "einhundert milliardste",
	"type_name": "ordinal"
}

{
	"start": 121,
	"end": 143,
	"resolution": {
		"value": "1000000300"
	},
	"text": "dreihundert milliardste",
	"type_name": "ordinal"
}
```